wordpress-mobile/WordPress-iOS
# [Bloggin Prompts] View prompt answers

Fixes #19891

To test:
1. Install & Launch JP. 
2. Enable "Blogging Prompts Enhancements" feature flag
3. Verify the "X Answers" button is green & tap on it
4. Verify **tag** screen on reader is opened with "dailyprompt-{id}" tag.
5. Verify an event with "blogging_prompts_other_answers_tapped" name is fired.
6. Disable the feature flag and verify the button is ineffective and is gray.

## Regression Notes
1. Potential unintended areas of impact
N/A

7. What I did to test those areas of impact (or what existing automated tests I relied on)
N/A

8. What automated tests I added (or what prevented me from doing so)
N/A

PR submission checklist:
